### ISSUE TITLE
Test function for long running processes

### ISSUE THREAD

**Issue by fabianmoronzirfas** [KWB-R/fhpredict#42]
I need a function to which I can send a number of milliseconds and that the function waits for that time.

like:
```plain
function test_delay(ms = 90000){
	delay(ms);
	return {success:true}
}
```

related to #27

**Comment by fabianmoronzirfas** [comment]
in #43 you only wait for 0.1 sec not for the functions parameter? @hsonne

**Comment by hsonne** [comment]
This should be solved since b6b959b.
